static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,64 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const ALLOWED_DOMAIN_EXTENSIONS = [".com", ".net"];
+const GITHUB_CLOUD = ["github.com", "www.github.com"];
+
+/**
+ * Method that checks the validity of the passed URL
+ *
+ * @returns {boolean}
+ */
+const checkValidUrl = inputURL => {
+	try {
+		const { protocol, hostname } = new URL(inputURL);
+
+		if (!ALLOWED_PROTOCOLS.some(allowedProtocol => protocol === allowedProtocol)) {
```

Comment:
/nit `some` is good if you need the function to compare, in this case I prefer to use a `includes`
e.g. `ALLOWED_PROTOCOLS.includes(protocol)`

Reply:
Updated